jobboard/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.conf import settings
+
+
+class Gender(models.TextChoices):
+    MALE = "M", "Male"
+    FEMALE = "F", "Female"
+    UNSPECIFIED = "U", "Unspecified"
+
+
+class Company(models.Model):
+    company_name = models.CharField(max_length=255)
+    company_description = models.TextField(max_length=500)
+    company_website_url = models.CharField(max_length=255)
+
+
+class Recruiter(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+    company_id = models.ForeignKey(Company, on_delete=models.RESTRICT, related_name='%(class)s_recruiters')
+    email = models.EmailField(max_length=255)
+    gender = models.CharField(max_length=1, choices=Gender.choices, default=Gender.UNSPECIFIED)
+    contact_number = models.IntegerField()
+
+
+class Student(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+    email = models.EmailField(max_length=255)
+    date_of_birth = models.DateField()
+    gender = models.CharField(max_length=1, choices=Gender.choices, default=Gender.UNSPECIFIED)
+    contact_number = models.IntegerField()
+    institute_university_name = models.CharField(max_length=255)
```

Comment:
Wouldn't it make sense to have a selectable list of universities? Do we expect each student to always belong to a single university? What about 2nd degree students? Or student that did more then one 1st degree? Or students that transferred?